pkg/utils/release/controller.go

```diff
@@ -39,50 +41,52 @@ func (s *SuiteController) CreateApplicationSnapshot(name string, namespace strin
 }
 
 // CreateRelease creates a new Release using the given parameters.
-func (s *SuiteController) CreateRelease(name, namespace, snapshot, sourceReleaseLink string) (*v1alpha1.Release, error) {
-	release := &v1alpha1.Release{
+func (s *SuiteController) CreateRelease(name, devNamespace, snapshot, sourceReleasePlan string) (*appstudiov1alpha1.Release, error) {
+	release := &appstudiov1alpha1.Release{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
-			Namespace: namespace,
+			Namespace: devNamespace,
 		},
-		Spec: v1alpha1.ReleaseSpec{
+		Spec: appstudiov1alpha1.ReleaseSpec{
 			ApplicationSnapshot: snapshot,
-			ReleaseLink:         sourceReleaseLink,
+			ReleasePlan:         sourceReleasePlan,
 		},
 	}
 
 	return release, s.KubeRest().Create(context.TODO(), release)
 }
 
-// CreateReleaseLink creates a new ReleaseLink using the given parameters.
-func (s *SuiteController) CreateReleaseLink(name, namespace, application, targetNamespace, releaseStrategy string) (*v1alpha1.ReleaseLink, error) {
-	releaseLink := &v1alpha1.ReleaseLink{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
-		},
-		Spec: v1alpha1.ReleaseLinkSpec{
-			DisplayName:     name,
-			Application:     application,
-			Target:          targetNamespace,
-			ReleaseStrategy: releaseStrategy,
-		},
-	}
+// GetRelease returns the release with the given name in the given namespace.
+func (s *SuiteController) GetRelease(releaseName, releaseNamespace string) (*appstudiov1alpha1.Release, error) {
+	release := &appstudiov1alpha1.Release{}
+
+	err := s.KubeRest().Get(context.TODO(), types.NamespacedName{
+		Name:      releaseName,
+		Namespace: releaseNamespace,
+	}, release)
 
-	return releaseLink, s.KubeRest().Create(context.TODO(), releaseLink)
+	return release, err
 }
 
 // CreateReleaseStrategy creates a new ReleaseStrategy using the given parameters.
-func (s *SuiteController) CreateReleaseStrategy(name, namespace, pipelineName, bundle string, policy string) (*v1alpha1.ReleaseStrategy, error) {
-	releaseStrategy := &v1alpha1.ReleaseStrategy{
+func (s *SuiteController) CreateReleaseStrategy(name, managedNamespace, pipelineName, bundle, policy, service_account string) (*appstudiov1alpha1.ReleaseStrategy, error) {
```

Comment:
I still think the parameter name should be namespace like it was. This function just creates a ReleaseStrategy where we tell it to. What if one day we want to create a release strategy in the dev workspace? There is no benefit to calling the function parameter like this IMO

Reply:
fixed